modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -395,9 +398,11 @@ function editConsentStatusFields($db, $user)
 {
     // Get CandID
     $candIDParam = $_POST['candID'];
-    $candID      = (isset($candIDParam) && $candIDParam !== 'null') ?
-        $candIDParam : null;
-
+    if (!isset($candIDParam) || $candIDParam === 'null') {
+        http_response_code(404);
```

Comment:
```suggestion
        http_response_code(400);
```
close! should be "Bad Request", not "Not Found"

Reply:
Done